ADOPTERS.md

```diff
@@ -35,3 +35,4 @@ This list is sorted in the order that organizations were added to it.
 | [Renault](https://www.renaultgroup.com/) | @smileisak | Building Renault Kubernetes Platform resources using XRDs and compositions for an additional layer of abstraction to provide end-user services. |
 | [Gympass](https://gympass.com) | @caiofralmeida | Builds a self-service platform so engineers can be more productive in resource provisioning. |
 | [Deutsche Kreditbank AG](https://www.dkb.de/) | @haarchri | Developer Self-Service for DKB Standard Operating Platform, higher standardization, simplification of tech stack and continuous reconciliation|
+| [ConsenSys](https://consensys.net/)           | @clementblaise | Main control plane for an internal developer platform|
```

Comment:
Is this IDP seeing production usage too? If so, it may be nice to explicitly mention that, e.g. "Main control plane for an internal developer platform being used in production" 🙏

Reply:
Thanks @jbw976 ! Our production only hosts development workloads for now, some should hit production in a few weeks. I will let you be the judge 😉

Reply:
i'll keep it honest for now hehe 😅 - feel free to keep us updated in the future when you progress your deployments!! 🙏